webmin-setup-repo.sh

```diff
@@ -106,6 +109,11 @@ detect_os() {
       install="$install_cmd -y"
       clean="yum clean all"
     fi
+  elif [ -n "$osid_suse_like" ]; then
+    package_type=rpm-md
+    install_cmd="zypper install"
+    install="$install_cmd -y"
+    clean="zypper clean."
```

Comment:
Why is there a period at the end of the command?

Reply:
Thanks for the pointer. I fixed that.